rosidl_runtime_c/include/rosidl_runtime_c/action_type_support_struct.h

```diff
@@ -36,6 +38,12 @@ struct rosidl_action_type_support_t
   const rosidl_message_type_support_t * status_message_type_support;
 };
 
+/// Macro to get the action type support.
```

Comment:
'Macro' here is unnecessary. Maybe "Get the action type support given a provided action and package"